single_tech_samples/storage/sample1_data_lifecycle/terraform/terraform.auto.tfvars

```diff
@@ -0,0 +1,22 @@
+resource_group_name       = "datalifecyclerg"
+location                  = "EastUS"
```

Comment:
General advise - "EastUS" is a busy region, suggest defaulting to a not-so-busy region for example "central india".

Reply:
Done